username_api.py

```diff
@@ -19,6 +23,50 @@
 # Discussion: github.com/manu-chroma/username-availability-checker/issues/79
 session = r.Session()
 
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+formatter = logging.Formatter('[%(levelname)s|%(filename)s:%(lineno)s] '
+                              '%(asctime)s %(message)s')
+handler.setLevel(logging.WARNING)
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+class DebugFilter(logging.Filter):
+    """
+        DebugFilter only logs if class variable DEBUG is True.
```

Comment:
need to follow pep257

See https://github.com/manu-chroma/username-availability-checker/issues/101 to fix the existing ones.